src/components/Header/Header.jsx

```diff
@@ -1,22 +1,25 @@
 import React from "react";
-import { Box, Stack, Typography } from '@mui/material';
+import { Box,Stack, Typography } from '@mui/material';
 import LocalPhoneIcon from '@mui/icons-material/LocalPhone';
 import MailIcon from '@mui/icons-material/Mail';
 
 function Header({orgInfo}) {
     return (
-        <Box sx={{ backgroundColor: '#000' }} width='100%' height='5vh'>
-            <Stack direction='row' justifyContent='flex-end' gap={2} marginRight='1rem' padding='3px' color='#fff'>
-                <Typography variant='subtitle1'><b>Reach us out here:  </b></Typography>
+        <Box sx={{ backgroundColor: '#000' }} display='flex' alignItems={'center'}  justifyContent={'flex-end'} width='100%' height='5vh' >
```

Comment:
Wrap this whole thing inside App Bar component and make the position sticky. Refer AppNavigation Bar Component.